src/transport_ofi.h

```diff
@@ -1213,4 +1213,63 @@ void shmem_transport_syncmem(void)
     // transport routine will be a nop until an API is provided.
 }
 
+static inline
+uint64_t shmem_transport_get_pending_put_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
+    SHMEM_TRANSPORT_OFI_CTX_LOCK(ctx);
+    cnt = SHMEM_TRANSPORT_OFI_CNTR_READ(&ctx->pending_put_cntr);
+    SHMEM_TRANSPORT_OFI_CTX_UNLOCK(ctx);
+    return cnt;
+}
+
+static inline
+uint64_t shmem_transport_get_pending_get_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
+    SHMEM_TRANSPORT_OFI_CTX_LOCK(ctx);
+    cnt = SHMEM_TRANSPORT_OFI_CNTR_READ(&ctx->pending_get_cntr);
+    SHMEM_TRANSPORT_OFI_CTX_UNLOCK(ctx);
+    return cnt;
+}
+
+static inline
+uint64_t shmem_transport_get_fi_put_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
+    SHMEM_TRANSPORT_OFI_CTX_LOCK(ctx);
+    cnt = fi_cntr_read(ctx->put_cntr);
+    SHMEM_TRANSPORT_OFI_CTX_UNLOCK(ctx);
+    return cnt;
+}
+
+static inline
+uint64_t shmem_transport_get_fi_get_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
+    SHMEM_TRANSPORT_OFI_CTX_LOCK(ctx);
+    cnt = fi_cntr_read(ctx->get_cntr);
+    SHMEM_TRANSPORT_OFI_CTX_UNLOCK(ctx);
+    return cnt;
+}
+
+static inline
+uint64_t shmem_transport_get_fi_target_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
+#if ENABLE_TARGET_CNTR
+#  ifdef USE_THREAD_COMPLETION
+    if (0 == pthread_mutex_lock(&shmem_transport_ofi_progress_lock)) {
+#  endif
+        cnt = fi_cntr_read(shmem_transport_ofi_target_cntrfd);
+#  ifdef USE_THREAD_COMPLETION
+        pthread_mutex_unlock(&shmem_transport_ofi_progress_lock);
+    }
+#  endif
+#else
+    cnt = -1;
```

Comment:
This should be 0, not -1 since -1 is not within the range of uint64_t.  If you want to return all bits set, better to use `UINT64_MAX`.